### ISSUE TITLE
Graph2Tree tutorial

### ISSUE THREAD

**Issue by maurovit** [graph4ai/graph4nlp#311]
Hello,
I reproduced your tutorial about Semantic Parsing using Graph2Tree architecture on Jobs Data. The problem is that the accuracy that i get after training is lower than the result written in tutorial: in fact, i get an accuracy of 85% on test set while on tutorial is written that the accuracy should increase until ~92%. Why this? Is something changed? How can arrive at this results?

Thank you in Advance

**Comment by AlanSwift** [comment]
Hi, @maurovit . Could you please provide the concrete information so we can reproduce it? (please refer to the [issue template](https://github.com/graph4ai/graph4nlp/blob/feature-add_pr_issue_template/.github/ISSUE_TEMPLATE/bug-report.md) )
We are sorry for the inconvenience.
@schenglee will follow up it.

**Comment by maurovit** [comment]
Hello everyone, first of all thank you for your reply and don't worry it's not a problem, i would just to know how i could optimize it.
However I followed your tutorial, I only changed the root directory in .yaml file and the GPU id in the args because the id of my gpu was 0 not 1. After that I used as data the data present in the repo at path graph2tree/jobs_data/raw. The accuracy that i get on test was around 85-86%. That's all i know. 😄

**Comment by schenglee** [comment]
@maurovit hi, it seems that this is caused by the word embedding matrix where the unknown token is not initialized correctly, [here](https://github.com/graph4ai/graph4nlp/blob/master/graph4nlp/pytorch/modules/utils/tree_utils.py#L253). 

I have fixed it and you can pull the code from the ``develop`` branch and retest it. 

Thank you for your feedback.

**Comment by maurovit** [comment]
Thank you, and congrats for your work 😄
